development/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM php:7.3-fpm-buster
+
+# Arguments defined in docker-compose.yml
+ARG user
+ARG uid
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+    git \
+    curl \
+    libpng-dev \
+    libonig-dev \
+    libxml2-dev \
+    zip \
+    unzip
+
+# Clear cache
+RUN apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# Install PHP extensions
+RUN docker-php-ext-install pdo_mysql mbstring exif pcntl bcmath gd \
+    && pecl install xdebug redis \
+    && docker-php-ext-enable xdebug redis \
+    && docker-php-source delete
```

Comment:
Do you have input regarding which of these dependencies are actually needed by the application?

As far as I know it does not use `gd` or `redis` so there is little value in including the extensions nor the related system dependencies.

Reply:
Has changed now. Extending the base image used by production.